File: unix/boot/spp/rpp/ratlibr/skipbl.r
include	defs

# skipbl - skip blanks and tabs at lin(i)

   subroutine skipbl(lin, i)
   character lin(ARB)
   integer i

   while (lin (i) == BLANK | lin (i) == TAB)
      i = i + 1

   return
   end

File: unix/boot/spp/rpp/ratlibf/skipbl.f
      subroutine skipbl(lin, i)
      integer lin(100)
      integer i
23000 if (.not.(lin (i) .eq. 32 .or. lin (i) .eq. 9))goto 23001
      i = i + 1
      goto 23000
23001 continue
      return
      end
